registration/tests/browser/browser_login.py

```diff
@@ -48,3 +48,18 @@ def test_invalid_credentials(self):
         self.fill_form({'username': self.username})
         self.assertIn('Please enter a correct username and password',
                       self.form.text)
+
+    def test_valid_credentials(self):
+        """
+        Check that we're logged in successfully when entering correct credentials.
+        """
+        self._login(
+            username=self.username,
+            password=self.password
+        )
+        # Logging in can direct a number of different ways, so we're checking that
+        # we _don't_ present an error message.
```

Comment:
Logging in a newly-created test user should always redirect to a consistent result page, so I don't see why you can't test that here? IMHO it would be better to test for the expected result than to test for the absence of one particular error message.